llvm/utils/release/build-docs.sh

```diff
@@ -89,28 +97,32 @@ if [ -n "$release" ]; then
   srcdir="./llvm-project/llvm"
 fi
 
+if [ "$no_doxygen" == "yes" ] && [ "$no_sphinx" == "yes" ]; then
+  echo "You can't specify both -no-doxygen and -no-sphinx, we have nothing to build then!"
+  exit 1
+fi
+
+if [ "$no_sphinx" != "yes" ]; then
+  sphinx_targets="docs-clang-html docs-clang-tools-html docs-flang-html docs-lld-html docs-llvm-html docs-polly-html"
+  sphinx_flag=" -DLLVM_ENABLE_SPHINX=ON -DSPHINX_WARNINGS_AS_ERRORS=OFF"
+fi
+
+if [ "$no_doxygen" != "yes" ]; then
+  echo "Doxygen: enabled"
+  doxygen_targets="$docs_target doxygen-clang doxygen-clang-tools doxygen-flang doxygen-llvm doxygen-mlir doxygen-polly"
+  doxygen_flag=" -DLLVM_ENABLE_DOXYGEN=ON -DLLVM_DOXYGEN_SVG=ON"
+else
+   echo "Doxygen: disabled"
+fi
```

Comment:
Why do you print a message for doxygen and not sphinx? I think we should be consistent here

Reply:
Good point! I added the `-no-sphinx` option later and just forgot. I will fix so that it always writes status for both.